app/viewmodels/DisplayMessage.scala

```diff
@@ -44,7 +44,7 @@ object DisplayMessage {
   case class CompoundMessage(message: DisplayMessage, next: DisplayMessage) extends InlineMessage
 
   implicit class DisplayMessageOps(message: DisplayMessage) {
-    def +(other: DisplayMessage): CompoundMessage = CompoundMessage(message, other)
+    def ++(other: DisplayMessage): CompoundMessage = CompoundMessage(message, other)
```

Comment:
this was required as sometimes the compiler is trying to use the `+` from scala String to add messages together